examples/plot_bids_features.py

```diff
@@ -0,0 +1,136 @@
+code and end with automatic publication that facilitates sharing and repproduction.
+Details about the BIDS standard can be consulted at http://bids.neuroimaging.io/
+We also demonstrate how to download individual groups of files from the
+Openneuro s3 bucket.
+
+More specifically:
+
+1. Download an fMRI BIDS dataset with derivatives from openneuro
+2. Extract automatically from the BIDS dataset first level model objects
+3. Demonstrate Quality assurance of Nistat estimation against available FSL estimation in the openneuro dataset
+4. Display contrast plot and uncorrected first level statistics table report
+
+Author : Martin Perez-Guevara: 2017
+"""
+
+import os
+from nilearn import plotting
+from scipy.stats import norm
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import nibabel as nib
+
+# from nistats.datasets import fetch_openneuro_dataset
+from nistats.first_level_model import first_level_models_from_bids
+from nistats.reporting import (
+    compare_niimgs, plot_contrast_matrix, get_clusters_table)
+from nistats.datasets import (fetch_openneuro_dataset_index,
+                              fetch_openneuro_dataset, select_from_index)
+from nistats.utils import get_design_from_fslmat
+
+##############################################################################
+# Fetch openneuro BIDS dataset
+# -----------------------------
+# We download one subject from an openneuro BIDS dataset.
+# We consider the stopsignal task from the ds000030 V4 dataset.
+# It contains the necessary information to run a statistical analysis using
+# Nistats. Also statistical results from an FSL analysis for an example QA.
+
+_, urls = fetch_openneuro_dataset_index()
+
+exclusion_patterns = ['*group*', '*phenotype*', '*mriqc*',
+                      '*parameter_plots*', '*physio_plots*',
+                      '*space-fsaverage*', '*space-T1w*',
+                      '*dwi*', '*beh*', '*task-bart*',
+                      '*task-rest*', '*task-scap*', '*task-task*']
+urls = select_from_index(
+    urls, exclusion_filters=exclusion_patterns, n_subjects=1)
+
+data_dir, _ = fetch_openneuro_dataset(urls=urls)
+
+##############################################################################
+# Obtain automatically FirstLevelModel objects and fit arguments
+# ---------------------------------------------------------------
+# From the dataset directory we obtain automatically FirstLevelModel objects
+# with their subject_id filled from the BIDS dataset. Moreover we obtain
+# for each model a dictionary with run_imgs, events and confounder regressors
+# since in this case a confounds.tsv file is available in the BIDS dataset.
+# To get the first level models we have to specify the dataset directory,
+# the task_label and the space_label as specified in the file names.
+# We also have to provide the folder with the desired derivatives, that in this
+# were produced by the fmriprep BIDS app.
```

Comment:
to be read again

Reply:
this will be retouched in the release PR

Reply:
'in this case' ?